data/human/deep jobs.txt

```diff
@@ -353,3 +353,617 @@ mission "Record academic conference"
+			`	As she catches her breath some more, her older colleague hustles over, bags in tow.`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"Where do you need to go?"`
+				`	"Why is it so urgent?"`
+				`	"Who are you?"`
+			`	Julie holds out her hand in greeting. "Apologies, let's do this properly. My name is Julie Gillar and my assistant and I were held back with our duties and missed our booked transport. We're in dire need of passage to Valhalla for several important business meetings, but unfortunately everything is booked up for the next five days straight, and that's just not acceptable to my employer."`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"Who's your employer?"`
+			`	Julie furtively glances at the people nearby. "Well, I have a few employers actually but at the moment, indirectly, Deep Security."`
+			`	She leans in slightly closer and lowers her voice. "We have access to their list of trustworthy pilots for emergencies. You'll be paid quite well."`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"I'm sorry, but I really can't help you right now."`
+					decline
+			label sure
+			`	"Lead right on then," says Julie.`
+				accept
+	on complete
+		log "Minor People" "Julie Gillar" `A consultant who worked with Deep Security. She requested emergency transport from you and showed you how to make yourself available to future consultants.`
+		payment 350000
+		set "deep security consultants"
+		conversation
+			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark she thanks you for your assistance. In addition to transferring <payment> into your account, she helps you change your job board preferences with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`
+			`	"Thank you Captain <last>, you've been most helpful. Do keep an eye out in the spaceports around here though. Deep Security is notorious for keeping us longer than we booked for. There's often someone stuck and in need of rapid transport off-world."`
+
+
+phrase "stranded consultant pickup"
+	word
+		`In accordance with the lodging request, you`
+		`Despite the short notice, you`
+	word
+		` `
+	word
+		`meet the consultants at their hotel accommodation`
+		`collect the consultants from the security station`
+		`fish the consultants out of a seedy bar`
+		`rescue the consultants from some overbearing companions`
+	word
+		` `
+	word
+		`in a quiet part of the spaceport`
+		`on the other side of town`
+		`in a raucous quarter just outside the port`
+	word
+		` `
+	word
+		`and escort them on board.`
+
+phrase "stranded consultant dropoff payment"
+	word
+		`The consultants thank you as you deliver them to their destination.`
+	word
+		` `
+	word
+		`You find out only afterwards that they left behind a soiled pair of boots.`
```

Comment:
```suggestion
		`You find out only afterward that they left behind a soiled pair of boots.`
```
Small American vs. British English thing. The general rule for these is take off the "s" for the American version (like toward vs. towards).